src/common/common.js

```diff
@@ -152,7 +152,7 @@ function cleanCommentData(comment, skipAdditional = []) {
       continue;
 
     case str.includes('* @param') || str.includes("* @return"):
-      str.replace(/(\s?\{.*?\})/, '')
+      str.replace(/(\s?\{.*?\}(\s|$))/, '$2')
```

Comment:
Would this be simpler

`str.replace(/(\s+\{.*?\}\s+)/, ' ')`

Reply:
I think I tried something like that at first, but there are a few @return statements that have no space after the return type (and no description either).